{{ cookiecutter.municipality_slug }}/python/example_scraper.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import random
+from datetime import datetime
+from typing import List
+
+from cdp_backend.pipeline.ingestion_models import (
+    Body,
+    EventIngestionModel,
+    EventMinutesItem,
+    Matter,
+    MinutesItem,
+    Person,
+    Role,
+    Seat,
+    Session,
+    SupportingFile,
+    Vote,
+)
+
+###############################################################################
+
+
+def get_example_events(
+    n: int = 10, is_minimal: bool = False
+) -> List[EventIngestionModel]:
+    return [_get_example_event(is_minimal) for _ in range(n)]
+
+
+###############################################################################
+
+
+FAKE_COUNCIL_SEAT_NUM = 10
+
+
+def _get_example_event(is_minimal: bool) -> EventIngestionModel:
+    "Create a fake example event data"
+    # Create a body for the event
+    body = Body(
+        name=f"Example Committee {random.randint(1, 100)}",
+        description="Example Description",
+    )
+    # Create sessions for the event
+    sessions = [
+        Session(
+            session_datetime=datetime.utcnow(),
+            video_uri="https://youtu.be/UjieH-NrOe8",
+        )
+        for _ in range(random.randint(1, 3))
+    ]
+    event_minutes_items = None
+    # Create event minutes items for the event
+    if not is_minimal:
+        # Get a number of event minutes items
+        event_minutes_items_num = random.randint(5, 15)
+        # Get a number of sponsors for each event minutes item
+        sponsors_nums = random.choices(range(1, 4), k=event_minutes_items_num)
+        # Specify the seat position for each sponsor
+        sponsors_seats = [
+            random.sample(range(1, FAKE_COUNCIL_SEAT_NUM + 1), sponsors_num)
+            for sponsors_num in sponsors_nums
+        ]
+        # Create a list of event minutes item
+        event_minutes_items = [
+            EventMinutesItem(
+                minutes_item=MinutesItem(
+                    name=f"Example Minutes Item {random.randint(1, 100)}",
+                    description="Example Description",
+                ),
+                matter=Matter(
+                    name=f"Example Matter {random.randint(1, 100)}",
+                    matter_type=f"Example Matter Type {random.randint(1, 5)}",
+                    title="Example Matter Title",
+                    sponsors=[
+                        _get_example_person(seat_num) for seat_num in sponsors_seats[i]
+                    ],
+                ),
+                supporting_files=[
+                    SupportingFile(
+                        name=f"Example Supporting File Name {file_num}",
+                        uri="https://www.w3.org/WAI/ER/tests/xhtml/testfiles/resources/pdf/dummy.pdf",
+                    )
+                    for file_num in range(1, random.randint(1, 5) + 1)
+                ],
+                decision=f"Example Decision {random.randint(1, 5)}",
```

Comment:
Hmmmm should we compute this based off individual votes? Would there ever be a situation where that mapping _isn't_ valid?

Reply:
Actually. I don't trust the math on our side to compute this. There are definitely special cases where a vote requires a super majority to pass and such and not just a simple majority. Having this field is better than having to bake in all those cases.

Reply:
Are you referring to `decision`? It just picks a random decision on the event minutes item

Reply:
Yea. I think this particular case should only have three possible which is "Passed", "Rejected", "Did not address" or something similar.

Reply:
I remember Emily and/ or Katlyn did some work in determine all the possible values for matters some where. Maybe pull those in?

Reply:
It was Adopted, Rejected, In Progress. I'll change it